front/app/components/admin/PostManager/CommonGroundInputManager/ImportInputsModal.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { Text, Box, Button } from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+import { IOption } from 'typings';
+
+import { IJob } from 'api/copy_inputs/types';
+import useCopyInputs from 'api/copy_inputs/useCopyInputs';
+
+import ProjectFilter from 'containers/Admin/reporting/components/ReportBuilder/Widgets/_shared/ProjectFilter';
```

Comment:
I think if we want to re-use this project filter, we might want to move it up into the shared component folder?